src/definitions_cxx.hpp

```diff
@@ -720,8 +720,9 @@ enum class GlobalMIDICommand {
 	LOOP_CONTINUOUS_LAYERING,
 	UNDO,
 	REDO,
+	FILL,
 };
-constexpr auto kNumGlobalMIDICommands = util::to_underlying(GlobalMIDICommand::REDO) + 1;
+constexpr auto kNumGlobalMIDICommands = util::to_underlying(GlobalMIDICommand::FILL) + 1;
```

Comment:
I'd suggest adding a LAST member to the enum and taking it's value, less likely to lead to mistakes down the road

Reply:
Agreed. This seemed like a very brittle way to do it in the original code. Maybe this is the first new global MIDI command to be added.

Reply:
Done 👍